config/config_test.go

```diff
@@ -10,25 +11,25 @@ import (
 	"go.viam.com/utils"
 )
 
-func TestValidate(t *testing.T) {
-	testCfgPath := "services.slam.attributes.fake"
-	logger := golog.NewTestLogger(t)
-
-	t.Run("Empty config", func(t *testing.T) {
-		model := resource.DefaultModelFamily.WithModel("test")
-		cfgService := resource.Config{Name: "test", API: slam.API, Model: model}
-		_, err := newConfig(cfgService)
-		test.That(t, err, test.ShouldBeError, newError("error validating \"services.slam.attributes.fake\": \"sensors\" must not be empty"))
-	})
-
+func testValidateTesthelper(
+	t *testing.T,
+	logger golog.Logger,
+	testCfgPath string,
+	cloudStoryEnabled bool,
+) {
```

Comment:
is there any way to get the `t.Run`s to include if the feature flag was enabled? 

My naive  approach being 
```
withFeatureFlag = ""
if cloudStoryEnabled{
   withFeatureFlag = "with cloudStoryEnabled"
}
t.Run("Simplest valid config" + withFeatureFlag, func(t *testing.T)
```

Reply:
oh yes definitely will add this in!